doc/source/user_guide/missing_data.rst

```diff
@@ -831,7 +831,7 @@ Operation        Result
 ================ ======
 ``pd.NA ** 0``   0
 ``1 ** pd.NA``   1
-``-1 ** pd.NA``  -1
+``-1 ** pd.NA``  pd.NA
```

Comment:
I think just remove this line. The sentence above describes this as "the result is known, even when one of the
operands is `NA`"